spec/requests/list_controller_spec.rb

```diff
@@ -609,20 +609,4 @@
       expect(topic_titles).to include(topic_in_sub_category.title)
     end
   end
-
-  describe "safe mode" do
```

Comment:
Is it possible to keep a test to ensure safe mode is still loading the proper javascript assets?

Reply:
I spent many hours to keep existing or create new spec tests for this PR. But it looks it's not possible since all the dynamic file creations are happening only on initial app booting time.